CHANGELOG.next.asciidoc

```diff
@@ -294,6 +295,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update PAN-OS fileset to use the ECS NAT fields. {issue}13320[13320] {pull}13330[13330]
 - Add fields to the Zeek DNS fileset for ECS DNS. {issue}13320[13320] {pull}13324[13324]
 - Add container image in Kubernetes metadata {pull}13356[13356] {issue}12688[12688]
+- Add timezone information to apache error fileset. {issue}12772[12772] {pull}13304[13304]
```

Comment:
Should this line be removed from this PR?

Reply:
Oops, my fault, I did the merge, let me fix it.

Reply:
Oh I see, I wanted to move the changelog entry :slightly_smiling_face: